gradle/scripts/README.md

```diff
@@ -0,0 +1,545 @@
+# Sensible multi-project defaults for Gradle
+
+The scripts here provides a simple way to configure a Java project with
+sensible defaults. By applying them, you can:
+
+- Manage dependencies using a simple YAML file.
+- Configure Checkstyle and JaCoCo code coverage.
+- Add [Javadoc offlink links](https://docs.oracle.com/javase/9/javadoc/javadoc-command.htm#GUID-51213F2C-6E01-4A03-A82A-17428A258A0F) easily.
+- Generate Maven BOM (Bill of Materials.)
```

Comment:
maybe (Bill of Materials).

Reply:
Thanks. Learned where a period should come: https://english.stackexchange.com/a/6634